libraries/c_sdk/standard/mqtt/README.md

```diff
@@ -1,18 +1,46 @@
 2. `MAX_NO_OF_MQTT_SUBSCRIPTIONS` - Maximum number of active MQTT subscriptions per MQTT connection. Default value is 10.
-3. `NETWORK_BUFFER_SIZE` - Static network buffer size for the MQTT packets to be sent using the MQTT shim. Please note that a different buffer is used for receiving MQTT packets and is dynamically allocated. Default value is 1024.
+3. `NETWORK_BUFFER_SIZE` - Static network buffer size for the MQTT packets to be sent using the Compatibility Layer. Please note that a different buffer is used for receiving MQTT packets. Default value is 1024.
 
 
-**Note:** The config `IOT_MQTT_ENABLE_SERIALIZER_OVERRIDES` to override serializer APIs is not supported in this MQTT shim implementation. [MQTT over BLE](https://docs.aws.amazon.com/freertos/latest/userguide/ble-demo.html#ble-demo-mqtt) is redesigned to avoid the dependency on this config by using a [BLE transport implementation](../ble/src/services/mqtt_ble/iot_ble_mqtt_transport.c).
+**Note:** The config `IOT_MQTT_ENABLE_SERIALIZER_OVERRIDES` to override serializer APIs is not supported in the Compatibility Layer. [MQTT over BLE](https://docs.aws.amazon.com/freertos/latest/userguide/ble-demo.html#ble-demo-mqtt) was dependent on this config. However, starting from [202011.00 release](https://github.com/aws/amazon-freertos/releases/tag/202011.00), MQTT over BLE is redesigned to avoid the dependency on this config by using a [BLE transport implementation](../ble/src/services/mqtt_ble/iot_ble_mqtt_transport.c). Please note that applications using MQTT over BLE with MQTT V2.x.x APIs will continue to work with this Compatibility Layer without having a dependency on the config `IOT_MQTT_ENABLE_SERIALIZER_OVERRIDES`.
+
+## Comparison of code sizes of the Compatibility Layer and the coreMQTT library
+
+Code sizes are calculated on [202011.00 release](https://github.com/aws/amazon-freertos/releases/tag/202011.00). Please note that only the unique dependencies of these libraries are accounted for in this comparison and common dependencies such as the FreeRTOS Kernel, are not considered. Hence, the comparison will only show the added cost to the memory when choosing one of these libraries for developing an MQTT application. Refer to the tables below for the comparison of code sizes of the **Compatibility Layer** and **coreMQTT library** with their unique dependencies.
+
+**Code sizes for Compatibility Layer and dependencies**
+| Modules | With -O1 Optimization | With -Os Optimization |
+| :-: | :-: | :-: |
+| Compatibility Layer | 15K | 13.5K |
+| coreMQTT | 6.9K | 5.7K |
+| Task Pool | 3.8K | 3.0K |
+| Platform Abstraction | 2.4K | 2.3K |
+| **Total estimate** | **28.1K** | **24.5K** |
+
+**Code sizes for coreMQTT library and dependencies**
+| Modules | With -O1 Optimization | With -Os Optimization |
+| :-: | :-: | :-: |
+| coreMQTT | 6.9K | 5.7K |
+| Transport implementation | 3.1K | 2.9K |
+| **Total estimate** | **10K** | **8.6K** |
```

Comment:
I'd suggest that we do not go into the details of the memory usage. Maybe we can just say that the compat layer uses X-KB, whereas coreMQTT Y-KB under similar settings. Separately, do users need to use the coreMQTT lib with the compat layer? If yes, I'd make it clear that for the end application, total memory usage is corMQTT + compat.

Reply:
Sounds good. I will make that change. Thanks